llmfoundry/data/packing.py

```diff
@@ -102,6 +101,18 @@ def pack(self, batch: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
         return batch
 
 
+def _trim_batch(
```

Comment:
docstring here would be nice too. took me a minute to figure out that this was trimming padding off of all examples